hathor/transaction/base_transaction.py

```diff
@@ -301,6 +301,24 @@ def calculate_height(self) -> int:
     def calculate_min_height(self) -> int:
         raise NotImplementedError
 
+    def calculate_feature_activation_bit_counts(self) -> Optional[list[int]]:
```

Comment:
Why do we need it on `BaseTransaction`? Why not have it only on `Block`?

Reply:
We don't, I just wanted to prevent `if self.is_block` conditionals in `BaseTransaction`, but I guess it's fine, I saw that it's used in other methods. Changed in https://github.com/HathorNetwork/hathor-core/pull/633/commits/3008ef370205679764c275db0ad2ba3087a9170c

Reply:
Ok. Thanks!